english/cycling-backwards/cycling-backwards.md

```diff
@@ -3,7 +3,16 @@ Unlike brown bears, who can only play balalaika, and only in Russia.
 
 ...but what about other animals that can cycle forwards.  [What do they do?](../cycling-forwards/cycling-forwards.md)
 
-[That's got to be horse...note to self.](../marshmallow.md)
+I sometimes pretty much think that only I can cycle in the forward direction. All the others ride in the backward directions. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.